langchain/src/storage/tests/convex/convex/langchain/db.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+export * from "langchain/util/convex";
```

Comment:
Remove?

Reply:
I think we want to keep the schema.ts and this file, so only `npx convex dev` is needed to get the test working?